site/src/pages/CreateTemplatePage/CreateTemplateForm.tsx

```diff
@@ -59,30 +67,31 @@ const getInitialValues = (starterTemplate?: TemplateExample) => {
 }
 
 interface CreateTemplateFormProps {
-  starterTemplate?: TemplateExample
-  error?: unknown
-  parameters?: ParameterSchema[]
-  isSubmitting: boolean
   onCancel: () => void
   onSubmit: (data: CreateTemplateData) => void
+  isSubmitting: boolean
   upload: TemplateUploadProps
+  starterTemplate?: TemplateExample
+  parameters?: ParameterSchema[]
+  variables?: TemplateVersionVariable[]
```

Comment:
In this case, you're adjusting the `CreateTemplateForm`. It seems that it has a similar logic to "edit template variables", but I'm afraid that it will hard to unify both forms?

Reply:
Yes! Because this form creates a template version + the template resource so it has some mixed values. I think this is a bit "nasty" too but Idk how to make it better or clear. 🤔 open to suggestions.

Reply:
This is a valid point, those forms are different indeed 👍